playground/templates/template.package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "dependencies": {
+    "@strapi/plugin-graphql": "${VERSION}",
+    "@strapi/plugin-i18n": "${VERSION}",
+    "@strapi/plugin-users-permissions": "${VERSION}",
+    "@strapi/strapi": "${VERSION}",
+    "better-sqlite3": "8.2.0",
+    "lodash.set": "^4.3.2",
+    "mime-types": "^2.1.27",
+    "strapi-plugin-init-admin-user": "^0.2.1"
+  },
+  "name": "playground",
+  "private": true,
+  "version": "0.1.0",
+  "description": "A Strapi application",
+  "scripts": {
+    "develop": "strapi develop",
+    "start": "strapi start",
+    "build": "strapi build",
+    "strapi": "strapi",
+    "reset": "node scripts/reset.js",
+    "cy:run": "cypress run",
+    "test": "echo no unittests",
+    "e2e": "ENV_PATH='./.env.test' start-server-and-test 'yarn start' http://0.0.0.0:1337 'yarn cy:run'"
```

Comment:
This is the only place where the port is still hard-coded, although I'm not sure if there is a nice way to fix this. Also, because of the env variable, e2e tests fail on windows.

Reply:
Don't thing we can do anything about the hardcoded stuff, windows is in this regard not necessary I'd say